content/deploy-troubleshooting.md

```diff
@@ -18,13 +18,13 @@ Note that code-level review lies outside the scope of Galaxy's support. If this
 
 <h2 id="503-errors">503 errors or Deployment Failures</h2>
 
-Your app may throw a 503 error and show `Service Unavailable: No healthy endpoints to handle the request` when you try to visit your URL. This means no healthy containers are currenty available to serve your app.
+Your app may throw a 503 error and show `Service Unavailable: No healthy endpoints to handle the request` when you try to visit your URL. This means no healthy containers are currently available to serve your app.
 
 There are several potential reasons for this. One example is that all containers are unhealthy, because they are all stuck in a CPU loop. Another reason is that none are running, because they all recently crashed (especially if the total number of your containers is 1, and it hasn't had time to restart). Another reason is because your build failed, if this is the first time you're deploying a container for that app or if the only other available containers built successfully but are unhealthy.
 
 The most common cause for this error is a problem in your code that prevents deployment.
 
-In some cases you may see the 503 error temporarily, if trying to resolve a URL in your browser within 1-2 minutes of issuing your deploy command. If you've waited 15 minutes or more, the problem will almost surely not resolve itself by waiting longer.
+In some rare cases you may see the 503 error temporarily, if trying to resolve a URL in your browser within 1-2 minutes of issuing your deploy command. Waiting should resolve this. If you've waited 15 minutes or more, however, the problem will almost surely not resolve itself by waiting longer.
```

Comment:
I just honestly don't know what you're talking about here.  If "deploys lead to downtime" is a behavior you're hearing about frequently from users, then that's a ginormous Galaxy bug and fixing it should be a high priority.